pkg/trace/api/otlp_test.go

```diff
@@ -87,14 +87,74 @@ var otlpTestTracesRequest = testutil.NewOTLPTracesRequest([]testutil.OTLPResourc
 	},
 })
 
+func TestRateKeys(t *testing.T) {
+	o := NewOTLPReceiver(nil, nil, config.New())
+	_, ok := o.rateKeys[serviceSignature{"a", "b"}]
+	require.False(t, ok)
+	o.rateKeys.Key("a", "b")
+	v, ok := o.rateKeys[serviceSignature{"a", "b"}]
+	require.True(t, ok)
+	require.Equal(t, "service:a,env:b", v)
+}
+
+func TestOTLPReceiverSamplingPriority(t *testing.T) {
+	mkspan := func(traceID uint64, service, env string) *pb.Span {
```

Comment:
`mkspan` doesn't actually use the `service` arg being provided here (although the tests seem to all pass as you only appear to use `a` for service name)

Reply:
Haha, doh. Thanks for catching that.